mitxpro/views.py

```diff
@@ -30,7 +31,7 @@ def get_js_settings_context(request):
         "public_path": public_path(request),
         "release_version": settings.VERSION,
         "recaptchaKey": settings.RECAPTCHA_SITE_KEY,
-        "sentry_dsn": sentry.get_public_dsn(),
+        "sentry_dsn": remove_password_from_url(os.environ.get("SENTRY_DSN", "")),
```

Comment:
Is a password used for the production sentry DSN?  I didn't notice one on CI and PR.

Reply:
Yeah there's one - https://github.com/mitodl/salt-ops/blob/master/pillar/heroku/xpro.sls#L120. This applies to all envs (ci, rc, and prod)

Reply:
There is not for mitxpro but I noticed it in micromasters. This is a precaution for when we adapt this code for micromasters and other projects with a legacy DSN value, so we don't leak the secret through the context